react-search/my-app/src/Items.tsx

```diff
@@ -0,0 +1,16 @@
+type ItemsProps = {
+  filteredQuotes: string[];
+};
+
+export function Items({ filteredQuotes }: ItemsProps) {
+  const items = [];
+  for (let i = 0; i < filteredQuotes.length; i++) {
+    items.push(<li key={i}> {filteredQuotes[i]} </li>);
+  }
+
+  if (filteredQuotes.length === 0) {
```

Comment:
It would be more efficient to do this check before the for-loop. With the code as it is, the for-loop is just wasted code.

Of course, efficiency considerations like this mean it will take 1.5 milliseconds to run instead of just 1.4 milliseconds. So it doesn't really matter. However, it makes the code cleaner to do the check first.